.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           ~/.cargo/registry
           ~/.cargo/git
           ~/.cargo/bin
-        key: rust-${{ runner.os }}-2020-11
+        key: rust-${{ runner.os }}-2021-06-25-01
```

Comment:
Could we set this value using a variable, with a comment, and then just update it in one place?

Reply:
Yep.